tests/ImageSharp.Tests/BootstrapperTests.cs

```diff
@@ -52,15 +52,15 @@ public void AddImageFormatGuardNull()
 
             exception = Assert.Throws<ArgumentNullException>(() =>
             {
-                Bootstrapper.AddImageFormat(null);
+                Bootstrapper.Default.AddImageFormat(null);
             });
 
             var format = new TestFormat();
             format.Decoder = null;
 
             exception = Assert.Throws<ArgumentNullException>(() =>
             {
-                Bootstrapper.AddImageFormat(format);
+                Bootstrapper.Default.AddImageFormat(format);
             });
             Assert.Contains("decoder", exception.Message);
```

Comment:
Is it really a robust solution to have this kind of "stringly typed" assertions? Instead of these long test cases wouldn't it be better to have multiple short AAA-style tests?